src/module/apps/characterImport/ArmorParser.js

```diff
@@ -62,7 +71,15 @@ export class ArmorParser {
 
         system.technology = parseTechnology(chummerArmor);
         system.description = parseDescription(chummerArmor);
-        
-        return createItemData(chummerArmor.name, 'armor', system);
+
+        // Assign import flags
+        system.importFlags = genImportFlags(formatAsSlug(chummerArmor.name_english), parserType);
```

Comment:
Here you use the english name instead of name. Is there a specific reason? For a user dabbling into writing macro or modules name matching would be easier

Reply:
I’m using the english name as I wanted to enable something more general down the road for icon assignment, where the name is matched to available icon names and chosen based on probability. This would be useful for some exotic weapons, bioware, etc. where the icons don’t perfectly match based on just the type & subtype.

If you think it’d be useful, I can set it to create both an English name slug and another id slug using the translated name?

Reply:
I dont care for either :) I was just thinking of macros and people which would make using the same reference easier